### PR TITLE
[minor] Conversion to "int" and "bool" for Doctrine generated entities

### PR DESCRIPTION
**Pull request by phansys**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Thanks for this commit. I like the idea behind it, but it looks like a hack. Apparently the code we are fixing is the same code that we generated (via Doctrine libraries). It's strange that we generate wrong code in the first place :confused:

**Comment by phansys** [comment]
Thank you for the feedback @javiereguiluz. I've created a PR last week against Doctrine in order to "fix" this into the library itself: doctrine/doctrine2#1457. I'm quoting fix because this CS rule isn't part of Doctrine's CSs:
* https://github.com/doctrine/doctrine2/blob/master/CONTRIBUTING.md#coding-standard
* https://github.com/doctrine/coding-standard/blob/master/Docs/README.md#doctrine-coding-standard

**Comment by javiereguiluz** [comment]
:+1: then it looks like this is the only safe way to solve this issue. Thanks.

File: Generator/DoctrineEntityGenerator.php [hunk 1]
```diff
@@ -88,6 +88,11 @@ public function generate(BundleInterface $bundle, $entity, $format, array $field
             $entityGenerator->setGenerateAnnotations(false);
             $entityCode = $entityGenerator->generateEntityClass($class);
         }
+        $entityCode = str_replace(
+            array("@var integer\n", "@var boolean\n", "@param integer\n", "@param boolean\n", "return integer\n", "return boolean\n"),
+            array("@var int\n", "@var bool\n", "@param int\n", "@param bool\n", "return int\n", "return bool\n"),
```
Comment (xabbuh) [inline Generator/DoctrineEntityGenerator.php]: Why didn't you put a `@` in front of `return`?

Comment (phansys) [inline Generator/DoctrineEntityGenerator.php]: I don't know @xabbuh :P (copy - paste typo). Fixed.

**Comment by javiereguiluz** [comment]
Thanks @phansys.

**Pull request merged**